### PR TITLE
Unit testing

### LINKED ISSUE #64: Old but still relevant documentation: Haxe unit testing
Here's the documentation from the old homepage: http://old.haxe.org/doc/cross/unit

### PR DESCRIPTION
**Pull request by markknol**
Based on http://old.haxe.org/doc/cross/unit
I left out the how-to-compile-this-thing part, since it doesn't require special setup.

Closes #64

### CODE REVIEW

File: 10-std.tex [hunk 1]
```diff
@@ -776,3 +776,57 @@ \subsection{Implementation details}
...
 This can lead to security issues in some cases. When in doubt, check the argument type when the function is called by using the \expr{Std.is} method.
+
+\section{Unit testing}
+\label{std-unit-testing}
+
+The Haxe Standard Library provides basic unit testing classes from the \type{haxe.unit} package. 
+
+\paragraph{Creating new test cases}
+
+Create a new class extending \type{haxe.unit.TestCase} and add own test methods. Every test method name must start with "\ic{test}".
```
Comment (Merelleya) [inline 10-std.tex]: First, create

File: 10-std.tex [hunk 2]
```diff
@@ -776,3 +776,57 @@ \subsection{Implementation details}
...
+
+\haxe{assets/UnitTestCase.hx}
+
+\paragraph{Running unit tests}
+To run the test, an instance of \type{haxe.unit.TestRunner} should be created. Add the \expr{TestCase} using the \expr{add} method and call \expr{run} to start the test.
```
Comment (Merelleya) [inline 10-std.tex]: should or "has to" (e.g. is it mandatory?)

File: 10-std.tex [hunk 3]
```diff
@@ -776,3 +776,57 @@ \subsection{Implementation details}
...
+Class: MyTestCase
+.
+OK 1 tests, 0 failed, 1 success
+\end{lstlisting} 
+
+\paragraph{Test functions}
+The \type{haxe.unit.TestCase} class comes with three test functions.
+
+\begin{description}
+	\item[\expr{assertEquals(a, b)}] Succeeds if \ic{a} and \ic{b} are equal, Where \ic{a} is value tested and \ic{b} is the expected value.
```
Comment (Merelleya) [inline 10-std.tex]: , where (no caps)

**Comment by markknol** [comment]
To confirm: Is it ready @Merelleya ?

**Comment by Merelleya** [comment]
all done!

**Comment by Simn** [comment]
Thanks!

**Pull request merged**